fix test_lazy_module_kwargs

avikchaudhuri opened the pull request:
Test Plan: fixed

Differential Revision: D64185644

tugsbayasgalan commented on torch/_functorch/aot_autograd.py:
```diff
@@ -1527,6 +1527,14 @@ def _detect_attribute_assignment(mod: torch.nn.Module):
         "_state_dict_pre_hooks",
         "training",
+    ]
+    NN_MODULE_LAZY_STD_ATTRS = [
```
Are they previously special handled in dynamo?

avikchaudhuri replied:
This only applies to non-strict, since this bug was in detecting arribute assigments (in general Dynamo errors so we don't run this detection there).

avikchaudhuri committed:
fix test_lazy_module_kwargs (#137705)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/137705

Test Plan: fixed

Reviewed By: tugsbayasgalan

Differential Revision: D64185644
